dockerfiles/centos/is-analytics/Dockerfile

```diff
@@ -38,8 +38,7 @@ ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}-${WSO2_SERVER_VERSION}
 
 # install required packages
 RUN yum check-update && \
-    yum install --downloadonly \
-    nc && \
+    yum install -y  nc && \
```

Comment:
what does -y correspond in here?

Reply:
-y is for yes
The most appropriate solution is to use yum install -y nc, where -y answers yes for all questions.